### PR TITLE
Updating minifier-js

### PR DESCRIPTION
**Pull request by denihs**
From this [issue](https://github.com/meteor/meteor/issues/11721) and [this](https://github.com/meteor/meteor/issues/11930) one, we find out a bug on terser (minify package) that was affecting some applications (more of that [here](https://github.com/terser/terser/issues/1155)). 

For that, we're replacing terser for [SWC](https://swc.rs/docs/configuration/minification).  With SWC we aren't just fixing those issues but also decreasing the minification time.

### CODE REVIEW

File: docs/history.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ Read our [Migration Guide](https://guide.meteor.com/2.7-migration.html) for this
 * `meteor-node-stubs@1.2.1`
   - Adds support for [node:](https://nodejs.org/api/esm.html#node-imports) imports.
 
+* `minifier-jss@2.8.0`
+  - Replacing
```
Comment (harryadel) [inline docs/history.md:88]: Missing swc?

Comment (harryadel) [reply docs/history.md:88]: https://github.com/terser/terser/blob/master/CHANGELOG.md#v5121
>Fixed an issue with function definitions inside blocks (https://github.com/terser/terser/issues/1155)

Comment (denihs) [reply docs/history.md]: My bad... I did a commit without adding the whole message. Thank you for pointing it out!

**Comment by denihs** [comment]
We encountered some challenges when trying to test the minify with SWC. We realized that the settings file used by the package wasn't being loaded correctly, so the code wasn't being minified all the way. On top of that, the minification time wasn't what we expected.

Our assumption that SWC was faster than terser was based on this [benchmark](https://github.com/privatenumber/minification-benchmarks), but what we got was (in average):

```
	time	size after minify
swc	7.869s	750.8 kB

terser
	4.742s	 462.8 kB
```

The test was made in a new app with some features. It's possible that the big difference is because of the missing settings, hard to say.

Based on all this, we decided to keep using terser for now, mainly because they released a new [version](https://github.com/terser/terser/blob/master/CHANGELOG.md#v5121) fixing the issues we're having.

I've already tested their solution, and it's working. I'll open a new PR updating terser to this new version soon.

Thanks to everyone who reviewed it.

**Pull request closed without merging**